library/alloc/src/collections/vec_deque/mod.rs

```diff
@@ -531,12 +531,13 @@ impl<T> VecDeque<T> {
     ///
     /// let deque: VecDeque<u32> = VecDeque::new();
     /// ```
-    // FIXME: This should probably be const
     #[inline]
     #[stable(feature = "rust1", since = "1.0.0")]
+    #[rustc_const_stable(feature = "const_vec_deque_new", since = "1.67.0")]
```

Comment:
You want the magic placeholder here, so we don't need to worry about updating it if it takes a while for hte PR to get approved:
```suggestion
    #[rustc_const_stable(feature = "const_vec_deque_new", since = "CURRENT_RUSTC_VERSION")]
```

Example:
https://github.com/rust-lang/rust/blob/ff0ffda6b3c3ea392c6cf88c676133666f491e5c/library/core/src/num/uint_macros.rs#L706